hack/charts/cluster-api-operator-providers/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: cluster-api-operator-providers
+description: Cluster API Provider Custom Resources
+type: application
+version: 0.0.0
+appVersion: "0.0.0"
+dependencies:
+  - name: cluster-api-operator
+    repository: file://../cluster-api-operator
```

Comment:
It probably won’t work outside of locally built chart. On the second thought, having charts separated would be simpler from release perspective

Reply:
Thx. I did it.